analyses/molecular-subtyping-ATRT/ATRT-molecular-subtyping.R

```diff
@@ -0,0 +1,275 @@
+# of the repository as follows:
+#
+# Rscript 'analyses/molecular-subtyping-ATRT/ATRT-molecular-subtyping.R'
+
+#### Set Up --------------------------------------------------------------------
+
+# Get `magrittr` pipe
+`%>%` <- dplyr::`%>%`
+
+#### Directories and Files -----------------------------------------------------
+
+# Detect the ".git" folder -- this will in the project root directory.
+# Use this as the root directory to ensure proper sourcing of functions no
+# matter where this is called from
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# File path to plots directory
+plots_dir <-
+  file.path(root_dir, "analyses", "molecular-subtyping-ATRT", "plots")
+
+if (!dir.exists(plots_dir)) {
+  dir.create(plots_dir)
+}
+
+# File path to results directory
+results_dir <-
+  file.path(root_dir, "analyses", "molecular-subtyping-ATRT", "results")
+
+if (!dir.exists(results_dir)) {
+  dir.create(results_dir)
+}
+
+# Define regions of the brain (using Anatomy of the Brain figure found at
+# https://www.ncbi.nlm.nih.gov/books/NBK65903/figure/CDR0000574573__205/)
+supratentorial <-
+  c(
+    "Skull",
+    "Thalamus",
+    "Temporal Lobe",
+    "Frontal Lobe",
+    "Parietal Lobe",
+    "Cerebrum",
+    "Basal Ganglia",
+    "Cranial Nerves NOS",
+    "Basal Ganglia;Temporal Lobe",
+    "Frontal Lobe;Parietal Lobe;Temporal Lobe",
+    "Parietal Lobe;Temporal Lobe",
+    "Frontal Lobe;Parietal Lobe",
+    "Frontal Lobe;Meninges/Dura;Parietal Lobe;Spinal Cord- Cervical;Spinal Cord- Lumbar/Thecal Sac;Spinal Cord- Thoracic;Temporal Lobe;Ventricles"
+    # Not sure how the line above should be broken up as `Frontal Lobe` is
+    # supratentorial, while `Spinal Cord` is not
+  )
+
+infratentorial <-
+  c(
+    "Cerebellum/Posterior Fossa",
+    "Brain Stem- Pons;Cerebellum/Posterior Fossa",
+    "Cerebellum/Posterior Fossa;Other locations NOS",
+    "Brain Stem",
+    "Brain Stem- Midbrain/Tectum;Ventricles",
+    "Cerebellum/Posterior Fossa;Ventricles",
+    "Cerebellum/Posterior Fossa;Spinal Cord- Cervical;Spinal Cord- Lumbar/Thecal Sac;Spinal Cord- Thoracic",
+    "Other locations NOS;Spinal Cord- Lumbar/Thecal Sac;Spinal Cord- Thoracic;Ventricles",
+    "Other locations NOS;Spinal Cord- Lumbar/Thecal Sac;Spinal Cord- Thoracic;Temporal Lobe"
+    # Not sure how the line above should be broken up as `Temporal Lobe` is
+    # supratentorial, while the `Spinal Cord` is not
+  )
+
+# Read in metadata
+metadata <-
+  readr::read_tsv(file.path(root_dir, "data", "pbta-histologies.tsv"))
+
+# Read in ssGSEA pathway information
+ssGSEA <-
+  as.data.frame(readr::read_rds(
+    file.path(
+      root_dir,
+      "analyses",
+      "ssgsea-hallmark",
+      "results",
+      "GeneSetExpressionMatrix.RDS"
+    )
+  ))
+
+# Read in RNA expression data
+stranded_expression <-
+  readr::read_rds(file.path(
+    root_dir,
+    "data",
+    "pbta-gene-expression-rsem-fpkm.stranded.rds"
+  ))
+
+# Read in focal CN data
```

Comment:
This needs a TODO about what focal CN data you'll be using because we'll get consensus files #128.